### ISSUE TITLE
Update creator to controller

### ISSUE THREAD

**Issue by OR13** [transmute-industries/PROPOSAL-OpenPgpSignature2019#4]
Section 3.3, Example 1:

> "creator": "https://example.com/jdoe/keys/1",

We've changed "creator" to "verificationMethod". You will also need to
determine the data format of the key that lives at
https://example.com/jdoe/keys/1 ... one thing we keep failing to do is
provide examples of key descriptions. Here's an Ed25519 one that you can
modify slightly and place into the spec:
```
{
  "id": "https://example.com/jdoe/keys/1",
  "type": "OpenPGPVerificationKey2018",
  "controller": "https://example.com/jdoe",
  "publicKeyPGP": "-----BEGIN PGP PUBLIC KEY BLOCK-----\r\n..."
}
```

**Comment by OR13** [comment]
https://w3c-dvcg.github.io/ld-signatures/

1.0 uses creator, until the next version is published, i think its best to stick with 1.0

**Comment by OR13** [comment]
done. https://github.com/transmute-industries/PROPOSAL-OpenPgpSignature2019/commit/7810df089d0d3b9e5d3f17a4f4e14405c75a33d3
